contributing/content-style.md

```diff
@@ -205,6 +206,30 @@ You should follow four rules | There are four rules that should be observed
 Your requests are redirected | The system redirects your requests
 This is caused by            | The cause of this issue has to do with
 
+## Use clear examples
```

Comment:
⛏️  It might be nice to add a note about the "variables in code" part, since IIRC sometimes we provide code (with variables) followed by examples.

Reply:
There's a link to the section in the Markup Style Guide. I figured the way it was done was more of a markup question rather than a content question.